docs/editors/emacs.md

```diff
@@ -116,15 +115,10 @@ To use Metals in Emacs, place this snippet in your Emacs configuration (for exam
 > You may need to disable other packages like `ensime` or sbt server to prevent
 > conflicts with Metals.
 
-If you start Emacs now then it will fail since the `metals-emacs` binary does
-not exist yet.
-
-```scala mdoc:bootstrap:metals-emacs emacs
-
-```
-
-The `-Dmetals.client=emacs` flag is important since it configures Metals for
-usage with Emacs.
+Next you have to install metals server. Emacs can do it for you when `lsp-mode`
+is enabled in a scala buffer or via `lsp-install-server` command. Also you can
+do it manually executing `coursier install metals` and configuring `$PATH`
```

Comment:
No, we don't need metals hacks for emacs anymore. Everything can be configured on `lsp-metals` side, so we can use generic metals.

Reply:
But you still mention `metals-emacs` above:
```
If you start Emacs now then it will fail since the `metals-emacs` binary does
￼not exist yet.
￼
￼```scala mdoc:bootstrap:metals-emacs emacs
￼
￼```
￼
￼The `-Dmetals.client=emacs` flag is important since it configures Metals for
￼usage with Emacs.
￼
```

Reply:
Yes, it's for those who prefer [eglot](https://github.com/joaotavora/eglot). I don't maintain nor use it and will leave for eglot users to decide how it should be installed.